src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -0,0 +1,144 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import SimpleSelect from '.';
+import Option from '../select-option';
+
+<Meta title="Design System/Simple Select" />
+
+# Simple Select
+
+
```

Comment:
I know we haven't got a template in place yet but do you think it would be worth adding a `Quick Start` section here with an example of how to import the component in to their project?

Reply:
good point, looks like I've forgot about that :)

Reply:
quick start section added

Reply:
Excellent - thank you. I've also noticed you've added a couple of other helpful sections to the MDX file too. Good thinking.